css-canvas-medical/index.html

```diff
@@ -0,0 +1,125 @@
+          <p class="hero-portion-text">Cant remember the last time you talked with a patient without focusing<br>
+            on the EMR? Canvas let you rediscover joy in the work you do. Capture<br>
+            meaningful data, identify gaps in care, and move seamlessly between<br>
+            tasks within the platform. You stay present with patients while your<br>
+            whole care team stays up-to-speed. Be a better doctor with Canvas,<br>
+            designed for primary care providers with the help of primary care<br> providers.
+          </p>
+        </section>
+        <input type="submit" name="get-started" value="Get Started">
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-full">
+        <section class="clinicians-prefer">
+          <h2>Clinicians Prefer Canvas</h2>
+          <p>Physician-owned primary care practices across the country have<br> already
+            made Canvas the fabric of their care.
+          </p>
+        </section>
+      </div>
+    </div>
+
+    <div class="row" id="locations">
+      <div class="column-half">
+        <p class="left-column">Cascade Family Practice, Portland OR <br> Temescal Creek Medicine, Oakland CA<br>
+          San Juan Healthcare, Friday Harbor WA
+        </p>
+      </div>
+      <div class="column-half">
+        <p class="right-column">Ward Medical Clinic, Bonifay FL<br> Caribou Medical, Towanda PA<br>
+          Bay Tree Family Care, Red Bay AL
+        </p>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-half">
+        <section class="learn-more">
+          <h2>Ready to learn more?</h2>
+          <p>Schedule a private meeting now</p>
+        </section>
+      </div>
+      <div class="column-half">
+        <input class="get-started" type="submit" name="learn-more-submit" value="Get Started">
+      </div>
+    </div>
+
+    <div class="row" id="about-us">
+      <div class="column-quarter">
+        <section>
+          <img class="logo-white" src="images/logo-white.png" alt="canvas-logo-white">
+          <p class="address">2307 Irving Street, Suite 228<br>San Francisco CA 94122<br>(800) 370-1416
+            <br>info@canvasmedical.com
+          </p>
+        </section>
+      </div>
+
+      <div class="column-quarter">
+      </div>
+
+      <div class="column-quarter">
+        <p class="company">Company</p>
+        <input class="about-us" type="submit" name="about-us" value="About Us">
+        <input class="blog" type="submit" name="blog" value="Blog">
+      </div>
+
+      <div class="column-quarter">
+        <p class="product">Product</p>
+        <input class="product-input" type="submit" name="product" value="Product">
+        <input class="implementation" type="submit" name="implementation" value="Implementation">
+        <input class="health-plans" type="submit" name="health-plans" value="For Health Plans">
+        <input class="security" type="submit" name="security" value="Security">
```

Comment:
These inputs should probably just be links. I think you'll be able to get rid of a lot of styling. Also, you can use `ul` for lists and use CSS to remove the bullets.